doc/source/whatsnew/v0.18.1.txt

```diff
@@ -131,6 +131,8 @@ These changes conform sparse handling to return the correct types and work to ma
 
 API changes
 ~~~~~~~~~~~
+- ``.swaplevel()`` for ``Series``, ``DataFrame``, ``Panel``, and ``MultiIndex`` now features defaults for its first two parameters ``i`` and ``j`` that swap the two innermost levels of the index.
```

Comment:
oh I just meant no need to add the line in between. its doesn't matter much as sphinx will ignore it, but just trying to be consistent (though I think there are some extra lines there anyhow....)